geospaas_harvesting/ingesters.py

```diff
@@ -121,11 +121,29 @@ def _ingest_dataset(self, url, normalized_attributes):
                 source=source)
 
             #Create the URI for the created Dataset in the database
-            _, created_dataset_uri = DatasetURI.objects.get_or_create(
+            _ , created_dataset_uri = DatasetURI.objects.get_or_create(
                 name=service_name,
                 service=service,
                 uri=url,
                 dataset=dataset)
+
+            for geos_parameter_id in range(len(geos_para_list)):
+                geos_parameter_info = geos_para_list[geos_parameter_id]
+                standard_name = geos_parameter_info.get('standard_name', None)
+                short_name = geos_parameter_info.get('short_name', None)
+                units = geos_parameter_info.get('units', None)
```

Comment:
You can use a simple, standard loop here.

(I also changed the variable names to match my other suggestions)
```suggestion
            for parameter_info in parameters_list:
                standard_name = parameter_info.get('standard_name', None)
                short_name = parameter_info.get('short_name', None)
                units = parameter_info.get('units', None)
```

Reply:
It is written in this way based on existence or non-existence of `short_name ` and `units ` for this parameters. This is what @akorosov told me to find the correct parameter of vocabulary by refining the search result of previous stage in a step by step manner.

If any other comments from @akorosov or @aperrin66 I am ready to do it.
@akorosov Please answer us in this regard.

Reply:
I wasn't talking about using the `get()` method of the dictionary, that's perfectly fine.
I meant the way the for loop is written: you don't need to use `range(len(...))` (see the change suggestion)

Reply:
So, OK. Now I got it. My badd. Actually, this comment of mine:
https://github.com/nansencenter/django-geo-spaas-harvesting/pull/17#discussion_r453575653
is related to this comment of yours:
https://github.com/nansencenter/django-geo-spaas-harvesting/pull/17#discussion_r453540797